test/functional/hosts_controller_test.rb

```diff
@@ -368,6 +368,17 @@ def setup_multiple_environments
     assert Host.find(@host2.id).environment == environments(:global_puppetmaster)
   end
 
+  test "user with edit host rights with update environments should change owner" do
```

Comment:
a) "with update owners"
b) it's in the middle of the two update environment tests, please move it below